index.bs

```diff
@@ -176,6 +181,24 @@ CONNECT request that initiated |session| is closed by the server, as described a
 [[!WEB-TRANSPORT-HTTP3]]
 [section 5](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-5).
 
+A [=WebTransport session=] has the following signals:
+
+<table class="data" dfn-for="session-signal">
+ <thead>
+  <tr>
+   <th>event
+   <th>definition
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td><dfn>DRAIN_WEBTRANSPORT_SESSION</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.6](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.6)
```

Comment:
https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.6 does not yet appear to define DRAIN_WEBTRANSPORT_SESSION. Do we need to wait on https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http3/pull/79 before merging this PR?